mantidproject/mantid
# Add manitdplot settings to workbench - part 2

~~**This PR should not be merged until #27787 has been merged**~~

**Description of work.**
This PR adds in mantidplot settings into workbench. This PR includes the 2 new tabs for fitting and plotting.

**To test:**
Open workbench
Click `file`->`settings`
Change the settings in the fitting and plotting tabs and check they work as expected. These tabs are new so all options need testing.

Fixes #27471 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
